reference/OpenAPI.json

```diff
@@ -23964,15 +23965,18 @@
           }
         }
       },
-      "8_res_list_reward_assignments": {
-        "title": "List Reward Assignments",
+      "LoyaltiesRewardsListAssignmentsResponseBody": {
+        "title": "Loyalties Rewards List Assignments Response Body",
         "type": "object",
-        "description": "Response body schema for **GET** `/loyalties/{campaignId}/rewards`.",
+        "description": "Response body schema for **GET** `v1/loyalties/{campaignId}/rewards`.",
```

Comment:
Musimy to wreszcie ustalić, jak to chcemy opisywać, czy z `v1`, czy bez `v1` :) Mi to jedno, ale może deweloperom to robi różnicę.

Jak uważasz? Jaki byłby lepszy dev exp. Twoim zdaniem?

Reply:
no ja bym dawał z v1 co wynika z tego ze inaczej nie da się zapisać client/v1/xxx :)

Reply:
no chyba ze chcesz raz zapisywać client/v1/xxx a raz xxx :)